example/android/app/src/main/AndroidManifest.xml

```diff
@@ -22,7 +22,7 @@
       <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
         <meta-data
           android:name="com.google.android.geo.API_KEY"
-          android:value="AIzaSyBAefhRlXEH3vCko-zZTX6PHllTR6av4WI"/>
+          android:value="AIzaSyAJVwyHn6klk0_aoEL1oEysTOYn0jrRU1c"/>
```

Comment:
Why? )) Does the old key work, doesn't it?

Reply:
Returned to old value